.circleci/config.yml

```
@@ -0,0 +1,85 @@
+# Use the latest 2.1 version of CircleCI pipeline process engine.
+# See: https://circleci.com/docs/2.0/configuration-reference
+version: 2.1
+
+# Settings common to each job
+var_job_defaults: &job_defaults
+  working_directory: ~/verdaccio-google-cloud-storage
+  docker:
+    - image: cimg/node:lts
+
+orbs:
+  node: circleci/node@5.1.0
+  build-tools: circleci/build-tools@3.0.0
+
+commands:
+  # Command for checking out the source code from GitHub. This also ensures that the source code
+  # can be merged to the master branch without conflicts.
+  checkout-and-rebase:
+    description: Checkout and verify clean merge with master
+    steps:
+      - checkout
+      - build-tools/merge-with-parent:
+          parent: master
+
+  setup:
+    description: 'Download the workspace content from the cache'
```

You don't need quotes.

Also, let's not use the term "cache" here, because caching implies something different in CircleCI terminology. See [Persisting data in workflows: when to use caching, artifacts, and workspaces][1] for more info.

I'd go with something like:
```suggestion
    description: Attack the workspace
```

[1]: https://circleci.com/blog/persisting-data-in-workflows-when-to-use-caching-artifacts-and-workspaces/

---

I believe you meant `Attach the workspace` 😄 

Changed

---

👊 🔫 ⚔️